fbinstant/utils/mock.lua

```diff
@@ -115,6 +115,12 @@ function fbinstant.get_supported_apis()
 	return {}
 end
 
+function fbinstant.get_sdk_version()
+	print("get_sdk_version")
+	return "6.1"
```

Comment:
FBInstant SDH has  getSDKVersion( ) method:
https://developers.facebook.com/docs/games/instant-games/sdk/fbinstant6.1
@britzl @subsoap

Reply:
This it the mock api to use when access to the real api isn't available.

Reply:
ok, sorry.

Reply:
np!